script.js

```diff
@@ -26,18 +26,98 @@ function createProductItemElement({ sku, name, image }) {
   return section;
 }
 
-function getSkuFromProductItem(item) {
-  return item.querySelector('span.item__sku').innerText;
-}
+const listOfProducts = async () => {
+  const response = await fetch('https://api.mercadolibre.com/sites/MLB/search?q=computador');
+  const { results } = await response.json();  
+  return results;
+};
+
+// function getSkuFromProductItem(item) {
+//   return item.querySelector('span.item__sku').innerText;
+// }
+
+const sumItemsPrices = async () => cartItemsList.reduce((acc, curr) => acc + curr.price, 0);
+const appendValue = async (sumReturn) => {
+  const createH2 = document.createElement('h2');
+  const selectDiv = document.getElementsByClassName('total-price')[0];
+  selectDiv.innerHTML = '';
+  createH2.innerHTML = await sumReturn;
+  selectDiv.appendChild(createH2);
+};
+
+const fetchCarItems = async (itemID) => {
+  const product = await fetch(`https://api.mercadolibre.com/items/${itemID}`);
+  const response = await product.json();
+  await appendValue(sumItemsPrices());
+  return response;
+};
 
 function cartItemClickListener(event) {
-  // coloque seu código aqui
-}
+  const getTargetId = event.target.id;
+  const response = fetchCarItems(getTargetId);
+  const getOL = document.getElementsByClassName('cart__items')[0];
+  getOL.removeChild(event.target);
+  cartItemsList.forEach((item, index) => {
+    if (item.id === getTargetId) cartItemsList.splice(index, 1);
+    localStorage.setItem('carShop', JSON.stringify(cartItemsList));
```

Comment:
```suggestion
    if (item.id === getTargetId) {
      cartItemsList.splice(index, 1);
      localStorage.setItem('carShop', JSON.stringify(cartItemsList));
    };
```
Colocar o localStorage dentro do if evita que ele seja salvo diversas vezes sem necessidade